docs/display.rst

```diff
@@ -0,0 +1,54 @@
+Display
+*******
+
+.. py:module:: microbit.display
+
+This module controls the 5×5 LED display on the front of your board. It can
+be used to display images, animations and even text.
+
+Functions
+=========
+
+.. py:function:: get_pixel(x, y)
+
+    Return the brightness of the LED at column ``x`` and row ``y``.
+
+
+.. py:function:: set_pixel(x, y)
+
+    Set the brightness of the LED at column ``x`` and row ``y``.
+
+
+.. py:function:: clear()
+
+    Set the brightness of all LEDs to 0.
+
+
+.. py:function:: print(string_or_image), delay=400)
+
+    If ``string_or_image`` is a string, it will be displayed one letter at a
+    time, with ``delay`` milliseconds delay between each pair of letters.  If
+    it is an ``Image``, it will simply show that image.  This function blocks
+    until it is finished.
```

Comment:
I changed the name of the parameter here, in an attempt to make it a little clearer, but this is a very unpythonic interface. Perhaps it would make sense to have a separate ``blit(image)`` (or ``show``) function.